scripts/setup/ruby.sh

```
@@ -63,15 +66,15 @@ install_bundler_if_needed() {
                                     echo "")
             
             if [ -n "$expected_version" ] && [ "$current_version" != "$expected_version" ]; then
-                log_warning "インストール済みのbundler ($current_version) と global-gems.rb で指定されたバージョン ($expected_version) が異なります"
-                log_info "既存のbundlerを使用して続行します"
+                echo "[WARN] "インストール済みのbundler ($current_version) と global-gems.rb で指定されたバージョン ($expected_version) が異なります""
```

Shellcheck (0.10.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)